src/main/java/com/example/demo/entity/Comment.java

```diff
@@ -0,0 +1,37 @@
+package com.example.demo.entity;
+
+import com.example.demo.dto.CommnetDTO;
+import lombok.*;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@EntityListeners(AuditingEntityListener.class)
+@Builder
+@Table(name = "COMMENT")
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long com_id;
+    private String comments;
+    @CreatedDate
+    private LocalDateTime createAt;
+    @LastModifiedBy
+    private LocalDateTime updatedAt;
+    @ManyToOne
+    @JoinColumn(name = "author_id", referencedColumnName = "user_account_id", foreignKey = @ForeignKey(value = ConstraintMode.NO_CONSTRAINT))
+    private UserAccount author;
+    private String body;
+
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
        @JoinColumn(foreignKey = @ForeignKey(value = ConstraintMode.NO_CONSTRAINT))
```

FK 제거. 어플리케이션 시작 시에 table, fk 전부 drop 하는 과정이 있는데, 이때 fk 가 없어서 에러가 발생할 수 있음